backend/tests/prisonerQuickLook.test.js

```diff
@@ -594,52 +600,562 @@ describe('prisoner profile quick look', () => {
+        expect.objectContaining({
+          visits: {
+            details: [
+              { label: 'Remaining visits', value: 'Unable to show this detail.' },
+              { label: 'Remaining privileged visits', value: 'Unable to show this detail.' },
+              { label: 'Next visit date', value: 'Unable to show this detail.' },
+            ],
+            visitSectionError: true,
+          },
+        })
+      )
+    })
+
+    it('should handle api errors for all data required for the schedules section', async () => {
+      await controller(req, res)
+
+      expect(res.render).toHaveBeenCalledWith(
+        'prisonerProfile/prisonerQuickLook/prisonerQuickLook.njk',
+        expect.objectContaining({
+          scheduledActivityPeriodsSectionError: true,
+        })
+      )
+    })
+
+    it('should load schedules without section error', async () => {
+      elite2Api.getEventsForToday.mockResolvedValue([
+        {
+          bookingId,
+          eventClass: 'INT_MOV',
+          eventStatus: 'SCH',
+          eventType: 'APP',
+          eventTypeDesc: 'Appointment',
+          eventSubType: 'EDUC',
+          eventSubTypeDesc: 'Education',
+          eventDate: '2020-04-17',
+          startTime: '2020-04-17T09:00:00',
+          endTime: '2020-04-17T10:00:00',
+          eventLocation: 'BADMINTON',
+          eventSource: 'APP',
+          eventSourceCode: 'APP',
+        },
+        {
+          bookingId,
+          eventClass: 'INT_MOV',
+          eventStatus: 'CANC',
+          eventType: 'APP',
+          eventTypeDesc: 'Appointment',
+          eventSubType: 'GYMSH',
+          eventSubTypeDesc: 'Gym - Sports Halls Activity',
+          eventDate: '2020-04-17',
+          startTime: '2020-04-17T15:00:00',
+          endTime: '2020-04-17T15:30:00',
+          eventLocation: 'BASKETBALL',
+          eventSource: 'APP',
+          eventSourceCode: 'APP',
+          eventSourceDesc: 'Test comment',
+        },
+        {
+          bookingId,
+          eventClass: 'INT_MOV',
+          eventStatus: 'SCH',
+          eventType: 'APP',
+          eventTypeDesc: 'Appointment',
+          eventSubType: 'GYMF',
+          eventSubTypeDesc: 'Gym - Football',
+          eventDate: '2020-04-17',
+          startTime: '2020-04-17T20:20:00',
+          endTime: '2020-04-17T20:35:00',
+          eventLocation: 'BADMINTON',
+          eventSource: 'APP',
+          eventSourceCode: 'APP',
+          eventSourceDesc: 'Testing a really long comment which is over 40 characters',
+        },
+      ])
+
+      await controller(req, res)
+
+      expect(res.render).toHaveBeenCalledWith(
+        'prisonerProfile/prisonerQuickLook/prisonerQuickLook.njk',
+        expect.objectContaining({
+          scheduledActivityPeriodsSectionError: false,
+        })
+      )
+    })
+
+    // it('should still render the quick look template with the correct error messages', async () => {
```

Comment:
Is this meant to be commented out?

Reply:
Yeah, it's going to be reinstated in an upcoming ticket.